service/src/test/java/com/ge/predix/controller/test/PolicyEvaluationControllerIT.java

```diff
@@ -104,12 +104,32 @@ public void setup() {
         this.notApplicableAndDenyPolicySets = createNotApplicableAndDenyPolicySets();
     }
 
+    @BeforeMethod
+    public void beforeMethod() {
+        MockSecurityContext.mockSecurityContext(this.testZone);
+        MockAcsRequestContext.mockAcsRequestContext(this.testZone);
+    }
+
     @AfterMethod
     public void testCleanup() {
         List<PolicySet> policySets = this.policyManagementService.getAllPolicySets();
         policySets.forEach(policySet -> this.policyManagementService.deletePolicySet(policySet.getName()));
     }
 
+    @Test
+    public void testPolicyZoneDoesNotExistException() throws Exception {
+        MockSecurityContext.mockSecurityContext(null);
+        MockAcsRequestContext.mockAcsRequestContext(this.testZone);
+
+        MockMvcContext postPolicyEvalContext = this.testUtils
+                .createWACWithCustomPOSTRequestBuilder(this.wac, this.testZone.getSubdomain(), POLICY_EVAL_URL);
+        postPolicyEvalContext.getMockMvc().perform(
+                postPolicyEvalContext.getBuilder().contentType(MediaType.APPLICATION_JSON)
+                        .content("testString")).andExpect(status().isBadRequest());
```

Comment:
How can we be sure that bad request status was indeed caused by ZoneDoesNotExistException?  We should mock exception throw.